### DISCUSSION TITLE
Thank you for the possibility of adding location on main page

### DISCUSSION THREAD

**Discussion by blogdemoi** [mcguirepr89/BirdNET-Pi#654] [General]
A great idea.

**Comment by blogdemoi** [comment]
However it seems there is a bug, as accents on letters, like "è" (in French), don't show. The letter is then replaced by nothing. I wanted to write "près" (near) and it shows "prs".

**Comment by ehpersonal38** [comment]
Currently it's set to remove all characters that are not a-z, 0-9, or a space. 

@lloydbayley is the only character that breaks the birdnet.conf formatting double quotes? If so we could possibly allow all special characters except that one.

**Reply by lloydbayley** [reply]
Single quotes are the worst but doubles should be disallowed too.
This was my lazy way of protecting things but hadn't thought about accented characters. Sorry @blogdemoi .
I had trouble with trying to match single and double quotes on their own. 
@ehpersonal38 If you are able to do a quick preg_replace formula that you know works, then please do replace the A-Z0-9

**Comment by blogdemoi** [comment]
> Sorry @blogdemoi .
> I had trouble with trying to match single and double quotes on their own.

It doesn't matter :-) and thank you for adding this improvment. Actually the city I live in doesn't use any accented characters. I wanted to add "près de Paris" (near Paris).

**Reply by lloydbayley** [reply]
Actually, I think I may have worked it out. Doing some testing.

**Comment by lloydbayley** [comment]
Right. Have tested extensively on my local install and it works. I've submitted a pull request with the changes in it.

**Reply by lloydbayley** [reply]
Sorry, sorry, ABORT ABORT... @ehpersonal38 has already done this and I wasn't aware.
I've reversed everything I just did now.

**Reply by ehpersonal38** [reply]
Yeah sorry about that, I should've let you know.

**Reply by lloydbayley** [reply]
It also could be said that I should've LOOKED! first! :)
It appears that even though I make sure I have fetched origin and such in Github Desktop, I still have to go to my repository on Github and click Sync Fork, otherwise I'm still working from the local copy. Bloody thing. Nevermind. Will remember for next time.
